onboard/nucleo_bridge/LCMHandler.cpp

```diff
@@ -0,0 +1,188 @@
+#include "LCMHandler.h"
+
+void LCMHandler::init() {
+    //Creation of lcm bus
+    lcm_bus = new lcm::LCM();
+    if (!lcm_bus->good())
+    {
+        printf("LCM Bus not created\n");
+        exit(1);
+    } else {
+	printf("LCM Bus created\n");
+    }
+    
+    internal_object = new InternalHandler();
+    
+    //Subscription to lcm channels 
+    lcm_bus->subscribe("/ik_ra_control", &LCMHandler::InternalHandler::ra_closed_loop_cmd, internal_object);
+    lcm_bus->subscribe("/sa_closedloop_cmd", &LCMHandler::InternalHandler::sa_closed_loop_cmd, internal_object);
+
+    lcm_bus->subscribe("/ra_openloop_cmd", &LCMHandler::InternalHandler::ra_open_loop_cmd, internal_object);
+    lcm_bus->subscribe("/sa_openloop_cmd", &LCMHandler::InternalHandler::sa_open_loop_cmd, internal_object);
+
+    lcm_bus->subscribe("/gimbal_openloop_cmd", &LCMHandler::InternalHandler::gimbal_cmd, internal_object);
+    lcm_bus->subscribe("/hand_openloop_cmd", &LCMHandler::InternalHandler::hand_openloop_cmd, internal_object);
+    lcm_bus->subscribe("/foot_openloop_cmd", &LCMHandler::InternalHandler::foot_openloop_cmd, internal_object);

```

Comment:
What is the reason for these groupings of 2, 2, and 3? if it's important add a comment. If not, maybe remove?